AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -579,7 +580,7 @@ - (void)openURLInAuthenticationSessionWith:(NSURL *)url API_AVAILABLE(ios(11)) {
     if (!strongSelf) {
       return;
     }
-    strongSelf.authenticationSession = nil;
+    [strongSelf clearAuthenticationSession];
```

Comment:
I think it makes more sense in reading the code if we break the strong reference at the end of the completion handler before `endBackgroundTask` instead of here.

Reply:
If we only break reference, we can have a chance to get 'Attempting to load the view of a view controller while it is deallocating'. Since the viewController still exists. According to the cancel documentation, it can fix our problem.

Reply:
No need to break the strong reference in the completion handler so resolving this conversation.